packages/edit-site/src/components/page-templates/style.scss

```diff
@@ -60,6 +60,7 @@
 
 	.dataviews-view-table & {
 		margin-bottom: $grid-unit-10;
+		display: block;
```

Comment:
The margin wasn't being applied causing alignment issues. Seems okay to sneak this fix in here?

Reply:
Yes.